.dev/docker/env/Dockerfile

```diff
@@ -36,44 +34,58 @@ RUN apt-get update \
       libusb-1.0-0-dev \
       libvtk${VTK_VERSION}-dev \
       libvtk${VTK_VERSION}-qt-dev \
+      lsb-release \
       qtbase5-dev \
       software-properties-common \
+      wget \
+      xvfb \
+ && if [ "$USE_LATEST_CMAKE" = true ] ; then \
+      apt-get -V install -y apt-transport-https ; \
+      wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null ; \
+      if [ "$(lsb_release -sr)" = "18.04" ]; then \
+        apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main' ; \
+      else \
+        apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main' ; \
+      fi \
+      && apt-get update ; \
```

Comment:
Doesn't apt-add-repository trigger an update automatically?

Reply:
Well, [since 18.04 yes](https://linuxize.com/post/how-to-add-apt-repository-in-ubuntu/#adding-repositories-with-add-apt-repository) - tried it now with 16.04 and there is indeed `apt-get update` is required. So we can indeed remove it now, since 16.04 is not anymore supported.